src/components/Discussion/index.jsx

```diff
@@ -1,20 +1,23 @@
-import { useState, useRef } from "react";
+import { useState, useRef, useEffect } from "react";
 import "./index.css";
 import Navbar from "../Navbar";
 import StarRating from "./rating";
 import Card from "./card";
+import Comment from "./editor";
+import { BiUserCircle } from "react-icons/bi";
 
 export default function Discussion() {
   const [comment, setComment] = useState("");
   const [name, setName] = useState("");
   const [count, setCount] = useState(0);
```

Comment:
why are you maintaining `count` state? Can't we use `commentList` to know the count